crates/backend/src/codegen.rs

```diff
@@ -745,7 +753,7 @@ impl TryToTokens for ast::Export {
                 #describe_args
                 #describe_ret
             },
-            attrs: attrs.clone(),
+            attrs: safe_attrs,
```

Comment:
I believe this line could be removed entirely, right?

Reply:
Oops, had to expand the code a bit to see what's going on here.
So yeah, ignore what I said above.